src/components/EditListingDescriptionPanel/EditListingDescriptionPanel.js

```diff
@@ -6,11 +6,15 @@ import { ensureOwnListing } from '../../util/data';
 import { findOptionsForSelectFilter } from '../../util/search';
 import { LISTING_STATE_DRAFT } from '../../util/types';
 import { ListingLink } from '../../components';
-import { EditListingDescriptionForm } from '../../forms';
+import { EditEquipmentListingGeneralForm, EditListingDescriptionForm } from '../../forms';
 import config from '../../config';
 
 import css from './EditListingDescriptionPanel.module.css';
 
+const LISTING_TYPE = {
+  equipment: 'equipment',
```

Comment:
This Listing_Type const was Duplicated in many files, why don't we declare it in only 1 constant file and export to other files?